algorithm_services/app.py

```diff
@@ -1,20 +1,53 @@
 import json
+import markdown
+import re
+import os.path
+import jinja2
 
 from flask import Flask
 from flask import render_template
 from flask import Response
+from flask import Markup
 
 from algorithm_services.algorithms.fizzbuzz import fizzbuzz
 from algorithm_services.algorithms.clock_angle import clock_angle
 
 
+def get_readme_algorithms_doc(readme_data):
```

Comment:
You should test this method. Only test your logic not the library; to mock libraries use mock patch: https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch